<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="PingException.xml" source-language="en-US" target-language="ja-JP">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5f431952bc2bd00792eaf22d97cfc26ac37339481.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">f431952bc2bd00792eaf22d97cfc26ac37339481</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Net.NetworkInformation.PingException">
          <source>The exception that is thrown when a <ph id="ph1">&lt;see cref="Overload:System.Net.NetworkInformation.Ping.Send" /&gt;</ph> or <ph id="ph2">&lt;see cref="Overload:System.Net.NetworkInformation.Ping.SendAsync" /&gt;</ph> method calls a method that throws an exception.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="Overload:System.Net.NetworkInformation.Ping.Send" /&gt;</ph> メソッドまたは <ph id="ph2">&lt;see cref="Overload:System.Net.NetworkInformation.Ping.SendAsync" /&gt;</ph> メソッドが、例外をスローするメソッドを呼び出したときにスローされる例外。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Net.NetworkInformation.PingException">
          <source>The <ph id="ph1">&lt;xref:System.Net.NetworkInformation.Ping&gt;</ph> class throws this exception to indicate that while sending an Internet Control Message Protocol (ICMP) Echo request, a method called by the <ph id="ph2">&lt;xref:System.Net.NetworkInformation.Ping&gt;</ph> class threw an unhandled exception.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Net.NetworkInformation.Ping&gt;</ph>クラスが、インターネット制御メッセージ プロトコル (ICMP) エコー要求を送信中にメソッドが呼び出されるを示すためには、この例外をスロー、<ph id="ph2">&lt;xref:System.Net.NetworkInformation.Ping&gt;</ph>クラスには、ハンドルされない例外がスローされました。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Net.NetworkInformation.PingException">
          <source>Applications should check the inner exception of a <ph id="ph1">&lt;xref:System.Net.NetworkInformation.PingException&gt;</ph> object to identify the problem.</source>
          <target state="translated">アプリケーションの内部例外を確認する必要があります、<ph id="ph1">&lt;xref:System.Net.NetworkInformation.PingException&gt;</ph>問題を識別するオブジェクト。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Net.NetworkInformation.PingException">
          <source>The <ph id="ph1">&lt;xref:System.Net.NetworkInformation.Ping&gt;</ph> class does not throw this exception if the ICMP Echo request fails because of network, ICMP, or destination errors.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Net.NetworkInformation.Ping&gt;</ph>ユーザーはネットワーク、ICMP、または変換先のエラーがあるため、ICMP エコー要求が失敗した場合は、クラスにはこの例外はスローされません。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Net.NetworkInformation.PingException">
          <source>For such errors, the <ph id="ph1">&lt;xref:System.Net.NetworkInformation.Ping&gt;</ph> class returns a <ph id="ph2">&lt;xref:System.Net.NetworkInformation.PingReply&gt;</ph> object with the relevant <ph id="ph3">&lt;xref:System.Net.NetworkInformation.IPStatus&gt;</ph> value set in the <ph id="ph4">&lt;xref:System.Net.NetworkInformation.PingReply.Status%2A&gt;</ph> property.</source>
          <target state="translated">このようなエラーの場合は、<ph id="ph1">&lt;xref:System.Net.NetworkInformation.Ping&gt;</ph>クラスを返します、<ph id="ph2">&lt;xref:System.Net.NetworkInformation.PingReply&gt;</ph>に関連するオブジェクト<ph id="ph3">&lt;xref:System.Net.NetworkInformation.IPStatus&gt;</ph>値で設定、<ph id="ph4">&lt;xref:System.Net.NetworkInformation.PingReply.Status%2A&gt;</ph>プロパティです。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="T:System.Net.NetworkInformation.PingException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Net.NetworkInformation.PingException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Net.NetworkInformation.PingException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.String)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the error.</source>
          <target state="translated">エラーを説明する <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Net.NetworkInformation.PingException" /&gt;</ph> class using the specified message.</source>
          <target state="translated">指定したメッセージを使用して、<ph id="ph1">&lt;see cref="T:System.Net.NetworkInformation.PingException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.String)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property with the text in the <ph id="ph2">`message`</ph> parameter.</source>
          <target state="translated">このコンス トラクター、<ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph>プロパティにテキストを含む、<ph id="ph2">`message`</ph>パラメーター。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.String)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property is set to <ph id="ph2">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> プロパティが <ph id="ph2">`null`</ph> に設定されている。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">シリアル化されたオブジェクト データを保持するオブジェクト。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that specifies the contextual information about the source or destination for this serialization.</source>
          <target state="translated">このシリアル化の転送元と転送先に関する文脈情報を指定する <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Net.NetworkInformation.PingException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">シリアル化したデータを使用して、<ph id="ph1">&lt;see cref="T:System.Net.NetworkInformation.PingException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute a previously serialized <ph id="ph1">&lt;xref:System.Net.NetworkInformation.PingException&gt;</ph> object.</source>
          <target state="translated">このコンス トラクターは、以前にシリアル化を再構成する逆シリアル化中に呼び出されます<ph id="ph1">&lt;xref:System.Net.NetworkInformation.PingException&gt;</ph>オブジェクト。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.String,System.Exception)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the error.</source>
          <target state="translated">エラーを説明する <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.String,System.Exception)">
          <source>The exception that causes the current exception.</source>
          <target state="translated">現在の例外の発生原因である例外。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Net.NetworkInformation.PingException" /&gt;</ph> class using the specified message and inner exception.</source>
          <target state="translated">指定したメッセージと内部例外を使用して、<ph id="ph1">&lt;see cref="T:System.Net.NetworkInformation.PingException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Net.NetworkInformation.PingException.#ctor(System.String,System.Exception)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property with the text in the <ph id="ph2">`message`</ph> parameter, and the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property with the <ph id="ph4">`innerException`</ph> parameter.</source>
          <target state="translated">このコンス トラクター、<ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph>プロパティのテキストに、<ph id="ph2">`message`</ph>パラメーター、および<ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph>を持つプロパティ、<ph id="ph4">`innerException`</ph>パラメーター。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>